docs/prerequisite_install_guide/prerequisite_install_guide.md

```diff
@@ -941,7 +941,7 @@ The first time that you launch a Visual Studio Emulator for Android device it wi
 
 > **Important:** You must be able to access the Android device on Windows in order for this to work.  It is outside of this guide to troubleshoot Windows not seeing your Android device.
 
-**WARNING:** With the default configuration of the Mobile App, you will not be able to communicate with the Api since it uses the web site running locally on your running using localhost which is not available to external devices or emulators.  To get around this you need to make sure that any Api changes you have made have have been submitted as a Pull Request and merged into the AllReady master branch.  
+**WARNING:** With the default configuration of the Mobile App, you will not be able to communicate with the Api since it uses the web site running locally on your running using localhost which is not available to external devices or emulators.  To get around this you need to make sure that any Api changes you have made have been submitted as a Pull Request and merged into the AllReady master branch.  
```

Comment:
I noticed that there is a double space between "emulators." and "To get around" in this paragraph. Would you mind fixing that up as part of this PR please?

Reply:
No problem. I'll look at that now.